tests/NuGet.Services.AzureSearch.Tests/Catalog2AzureSearch/Integration/InMemoryCloudBlobContainer.cs

```diff
@@ -30,7 +31,7 @@ public Task<bool> DeleteIfExistsAsync()
             throw new NotImplementedException();
         }
 
-        public Task<bool> ExistsAsync()
+        public Task<bool> ExistsAsync(BlobRequestOptions options = null, OperationContext operationContext = null)
```

Comment:
This change is required as the `ExistsAsync` API on the `NuGetGallery.ICloudBlobContainer` interface changed.